packages/api-reference/src/components/Content/Schema/SchemaObjectProperties.vue

```diff
@@ -106,8 +106,8 @@ const getAdditionalPropertiesValue = (
         schema.discriminator?.propertyName || discriminatorPropertyName
       "
       :isDiscriminator="
-        property ===
-        (schema.discriminator?.propertyName || discriminatorPropertyName)
+        property === schema.discriminator?.propertyName &&
+        !!schema.discriminator
```

Comment:
too restrictive, now it doesn't apply at all in nested level which can still hold discriminator